ui/components/ui/info-tooltip/info-tooltip.stories.js

```diff
@@ -17,7 +17,6 @@ export default {
       control: 'select',
       options: ['top', 'left', 'bottom', 'right'],
     },
-    wide: { control: 'boolean' },
     containerClassName: { control: 'text' },
     wrapperClassName: { control: 'text' },
     iconFillColor: { control: 'text' },
```

Comment:
Used once and removed. See comment on `ui/components/ui/info-tooltip/info-tooltip.js`